public/app/plugins/datasource/cloudwatch/components/MetricsQueryEditor/MetricsQueryEditor.tsx

```diff
@@ -130,17 +131,37 @@ export const MetricsQueryEditor = (props: Props) => {
           />
         </EditorField>
 
-        <EditorField
-          label="Alias"
-          width={26}
-          optional
-          tooltip="Change time series legend name using this field. See documentation for replacement variable formats."
-        >
-          <Alias
-            value={preparedQuery.alias ?? ''}
-            onChange={(value: string) => onChange({ ...preparedQuery, alias: value })}
-          />
-        </EditorField>
+        {config.featureToggles.cloudWatchDynamicLabels ? (
+          <EditorField
+            label="Label"
+            width={26}
+            optional
+            tooltip="Change time series legend name using Dynamic labels. See documentation for details."
```

Comment:
it should be possible to use a HTML element in the tooltip so you can add a link to the documentation here.

Reply:
Yes good point. Will do this when we have created section about this in the docs. It's the last sub task in the epic.